src/sample-data/sample-data-generator.ts

```diff
@@ -54,7 +54,7 @@ export class SampleDataGenerator {
                     let p = new WebPage(data);
 
                     //Check if page is in english
-                    LanguageExtractor.isWebPageInLanguage(p, LanguageExtractor.ENGLISH_LANG_CODE, function (result: boolean) {
+                    LanguageExtractor.isWebPageInLanguage(p, "en", function (err, result) {
```

Comment:
Does it work if I accidentally pass "eng" as a parameter? Something like that happened to me recently, that's why I added ENGLISH_LANG_CODE